### PR TITLE
update documentation to reflect new developer workspace

### PR DESCRIPTION
**Pull request by rossiam**
Updated the READMEs for the examples to reflect the new developer workspace.

This closes issue #36 .

### CODE REVIEW

File: examples/java-lambda-smartapp/README.md [hunk 1]
```diff
@@ -89,19 +89,16 @@ First, you'll need to determine the target ARN for the function you just deploye
...
-|--------------------|-------------------------------------|
-| SmartApp Instances | Single                              |
-| Hosting Type       | AWS Lambda                          |
-| Target ARN         | The ARN you found above             |
-| Scopes             | `r:devices:*` and `x:devices:*`     |
-| App Settings       | leave blank                         |
-
-Once created, self-publish the automation for testing and you will be ready to test it.
+Then, follow these steps using the
+[developer workspace](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html):
```
Comment (erodewald) [inline examples/java-lambda-smartapp/README.md]: The new URL appears to be https://smartthings.developer.samsung.com/workspace/

File: examples/java-ratpack-guice-smartapp/README.md [hunk 2]
```diff
@@ -20,21 +22,24 @@ Execute the following command from the project root directory:
...
-| Hosting Type       | WebHook endpoint                    |
-| Target URL         | Enter the URL of your endpoint here |
-| Scopes             | `r:devices:*` and `x:devices:*`     |
-
-Once created, self-publish the automation for testing and you will be ready to test it.
+Note: it is necessary for the application to be running before you create the
+SmartApp in the developer console. You will need to restart once configured.
+
+Follow these steps using the
+[developer workspace](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html):
```
Comment (erodewald) [inline examples/java-ratpack-guice-smartapp/README.md]: same here -> https://smartthings.developer.samsung.com/workspace/

File: examples/java-springboot-smartapp/README.md [hunk 3]
```diff
@@ -20,26 +20,28 @@ Execute the following command from the project root directory:
...
-
-### Configure Public Key
-
-Copy the public key created in the previous step into a file called `smartthings_rsa.pub` in `src/main/resources`
-and restart your server (see [Build and Run](#build-and-run) above).
+Note: it is necessary for the application to be running before you create the
+SmartApp in the developer console. You will need to restart once configured.
+
+Follow these steps using the
+[developer workspace](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html):
```
Comment (erodewald) [inline examples/java-springboot-smartapp/README.md]: also here https://smartthings.developer.samsung.com/workspace/

File: examples/kotlin-smartapp/README.md [hunk 4]
```diff
@@ -16,21 +18,24 @@ From the root project directory:
...
-| Hosting Type       | WebHook endpoint                    |
-| Target URL         | Enter the URL of your endpoint here |
-| Scopes             | `r:devices:*` and `x:devices:*`     |
-
-Once created, self-publish the automation for testing and you will be ready to test it.
+Note: it is necessary for the application to be running before you create the
+SmartApp in the developer console. You will need to restart once configured.
+
+Follow these steps using the
+[developer workspace](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html):
```
Comment (erodewald) [inline examples/kotlin-smartapp/README.md]: https://smartthings.developer.samsung.com/workspace/

File: examples/java-lambda-smartapp/README.md (hunk 1 above)
Comment (rossiam) [reply examples/java-lambda-smartapp/README.md]: Fixed...as well as the others.

**Pull request merged**